skins/laika/src/components/pages/donation_form/AddressType.vue

```diff
@@ -41,13 +42,28 @@ export default Vue.extend( {
 			type: AddressTypeModel.PERSON,
 		};
 	},
+	props: {
+		isDirectDebit: Boolean,
+	},
 	computed: {
 		AddressTypeModel: {
 			get: function () {
 				return AddressTypeModel;
 			},
 		},
 	},
+	watch: {
```

Comment:
Don't know if this was the right approach.

Reply:
It's a bit weird that the relatively self-contained address type now knows about payment types (at least at a semantic level). Could you have a property `disabledTypes` instead, that is an empty array by default. Then the parent (`Address`) component would be responsible to generate the right types (empty array or array with `AddressTypeModel.ANON`). This more generic approach will help us when we tackle the new "EMAIL" address type (https://phabricator.wikimedia.org/T220367)

Now that I think of it, this kind of "business logic" probably belongs in a store getter.